Documentation/Signoffs/Camera_Power_Signoff.md

```diff
@@ -0,0 +1,136 @@
+
+## Buildable Schematic
+
+![PDA_Wiring_Schematic](https://user-images.githubusercontent.com/80428236/219553780-1b5e38c1-d0f1-47f3-a0a0-48e2fc9916c7.png)
+
+Figure 1. Electrical Wiring Schematic
+
+## Analysis
+
+Selecting the Power Source
+
+1. It serves to the convenience of this subsystem that all devices that need to be powered can accept or require 12 V to operate (as seen in the constraints section). Therefore, it has been decided to go with a 12 V 7 Ah LiFePO4 battery produced by Howell Energy [6]. These batteries do not occupy much space (5.94 x 2.56 x 3.7 in) and last much longer than their sealed lead-acid counterpart, as well as being capable of being recharged [6]. 
+
+Recharging the Power Source
+
+1. The only feasible solution for recharging the battery given its location is by using a solar panel. Therefore, a 30 W 24 V solar panel made by Newpowa [7] has been selected. The solar panel will be used in tandem with a charger controller designed to take in 16-25 V and convert it to a 12-14.6 V output [9], and is also rated to handle up to 150 W so there should be no compatibility issues. The charge controller is also designed in mind for charging LiFePO4 batteries [9], so it should function well with the Howell battery.
+
+Connecting the Devices
+
+1. Cameras
+    * Given that the wifi module for this subsystem will take the ethernet port located on the camera, the 2-pin terminal block will have to be used. Therefore, wire will just be crimped between the camera and battery with an intermediary 750 mA fuse [12] in order to protect it in the case of overcurrent.
+2. Wifi Module
+    * It has already been decided in the communication subsystem for primary data acquisition that the USB power supply will remain connected. Given that the USB connection looks for a 5 V input [2], it has been decided to use a step down converter from 12 V to 5 V which can handle a maximum current of 3 A [10]. This should be compatible given the wifi module will draw a maximum of 2 A [2], and it will be protected with 2.5 A fuse [13] to prevent overcurrent.
+3. Arduino Nano
+    * Given that the Arduino board only allows the option of using the Vin and GND pins for an external power source [3], wire from the battery to the Arduino pins will be used to connect to power. To protect the Arduino from overcurrent, 250 mA fuses [11] will be connected between the Arduino and battery.
+4. Heat Plate
+    * The heat plate only has a positive and negative wire, so it will be connected via crimping to the battery. In order to protect from overcurrent, 250 mA fuses [11] will be used.
+
+Distributing Power
+
+1. The camera, wifi module, Arduino, and heat plate draw roughly 500 mA, 2 A, 85 mA, and 200 mA, respectively, as seen in the constraints. Summed up, this equals 2.785 A of current drawn which is far less than the max 7 A outputted by the Howell battery [6], so there is no concern about making sure all devices get sufficient power.
```

Comment:
Why are there so many fuses? and they are not in the design, so it is not buildable.